src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets

```diff
@@ -326,7 +326,18 @@
     </ItemGroup>
 
     <WriteItemsToJson JsonFileName="%(HelixWorkItemList.BuildCompleteJson)" Items="@(BuildComplete)" />
-    <Message Text="Wrote build complete JSON for @(BuildComplete->Count()) Queues." />
+    <Message Text="Wrote job-start (build complete) JSON for @(BuildComplete->Count()) Queues." />
+	
+    <CreateItem Include="%(HelixWorkItemList.BuildCompleteJson)" AdditionalMetadata="RelativeBlobPath=JobStartJsonMessages.json">
+      <Output TaskParameter="Include" ItemName="JobStartJsons" />
+    </CreateItem>
```

Comment:
Creates a new ITaskItem - using it here to add metadata at the same time, since the item I'm starting from is metadata itself.    The Upload task requires there to be some RelativeBlobPath  present.